config_defaults.yml

```diff
@@ -29,13 +29,25 @@
     env: PUPPET_GEM_VERSION="~> 3.0"
     bundler_args: --without system_tests
 Gemfile:
+  supports_windows:
+    - puppetlabs-acl
+    - puppetlabs-chocolatey
+    - puppetlabs-dism
+    - puppetlabs-dsc
+    - puppetlabs-powershell
+    - puppetlabs-reboot
+    - puppetlabs-registry
+    - puppetlabs-sqlserver
+    - puppetlabs-wsus_client
```

Comment:
@DavidS Yes! It should go away entirely; see @ferventcoder's and my comment as such at https://github.com/puppetlabs/modulesync_configs/pull/99#discussion_r86820942